### ISSUE TITLE
Is it possible to use a model trained on huggingface transformer for getting saliency scores?

### ISSUE THREAD

**Issue by fabrahman** [successar/FRESH#5]
Hi,

I wonder if I can use a model trained on huggingface transformer library and get the saliency scores for the token in my dataset? Can I simply modify the code to pass `pytorch_model.bin` as the `args.archive_file` instead of tar.gz?
I would appreciate if you could please point me to the right direction.

**Comment by ZexueHe** [comment]
I have the same question. Did you solve it?

**Comment by successar** [comment]
Hi

If you just need gradients, I would suggest using captum.ai library which provides easy access to those. Currently my code modules depend on each other a lot and might end up need a bunch of extra work just to get gradients.
